perception/autoware_object_merger/src/object_association_merger_node.cpp

```diff
@@ -224,6 +252,49 @@ void ObjectAssociationMergerNode::objectsCallback(
     }
   }
 
+  // Remove overlapped known object
```

Comment:
1. Please integrate the two pieces of code; unknown objects and known objects now have similar coding structures. Please integrate them together.
2. Regarding efficiency, please optimize the looping, for now, all objects will be compared twice.

Reply:
Generally similar, but there are some minor differences that may not be as significant as the benefits of putting them together.
First, sorting improves code efficiency.